patterns/nvidia-gpu-efa/README.md

```diff
@@ -82,84 +88,178 @@ See [here](https://aws-ia.github.io/terraform-aws-eks-blueprints/getting-started
     clusterrole.rbac.authorization.k8s.io/mpi-operator configured
     ```
 
-3. EFA test
+3. EFA info test
 
-    The results should shown that two EFA adapters are available (one for each worker pod)
+    This test prints a list of available EFA interfaces by using the `/opt/amazon/efa/bin/fi_info` utility.
+    Edit the script [generate-efa-info-test.sh](generate-efa-info-test.sh) and adjust the following environment variables if needed prior to running it:
+    
+    ```sh
+    export NUM_WORKERS=2
```

Comment:
the pattern is static so we don't use variables - we can educate and explain how users should adjust based on the instance type, and its subsequent resources, but the solution itself is static

Reply:
I think the script makes sense here, but I agree that the user should not have to edit variables. 
I modified the instructions. Just by running the script, the default values will work. Users can easily adapt the example to other scenarios by setting different values later.